js/web/test/suite-test-list.jsonc

```diff
@@ -2365,14 +2372,15 @@
       // "test_sinh",
       // // "test_size_example",
       // // "test_size",
-      "test_slice_default_axes",
-      "test_slice_default_steps",
-      "test_slice_end_out_of_bounds",
-      "test_slice_neg_steps",
-      "test_slice_neg",
-      "test_slice_negative_axes",
-      "test_slice_start_out_of_bounds",
-      "test_slice",
+      // tests "test_slice_*" on opset > 9 are excluded because starts, ends, axes and steps are non-constant inputs.
```

Comment:
In practice they are constant inputs for most models. Even if the input tensor itself is technically *capable* of being dynamic, ORT almost always resolves them to constants via constant folding. So I wonder if the issue is that this test case binds the input dynamically. Anyway, keep as-is - just seeking understanding.

Reply:
These unit tests make starts, ends, axes, etc. as non-constant inputs, i.e. we can only get their values during inference.
But WebNN `slice` needs to know the starts and sizes for operand construction, that's why they'll fallback.

<img width="714" height="398" alt="image" src="https://github.com/user-attachments/assets/5b051467-6a6e-4805-a775-3ad74fb80369" />